charts/performancetesting/values.yaml

```diff
@@ -0,0 +1,20 @@
+
+replicaCount: 1 #This is not used for now
+namespace: dev
+schedule: "0 9 * * 1" #It runs At 9:00 every Monday
```

Comment:
We'll likely change this schedule on an environment basis but, just in case, could you consider changing this to something that doesn't happen at the start of the workday? In case system load effects daily development?

Maybe `0 1 * * 1` # Runs at 1AM every Monday

Reply:
It looks great! Actually this was my question for the team.